zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -640,14 +725,41 @@ pub(crate) fn put_transparent_output<P: consensus::Parameters>(
         ":address": &address.encode(params),
         ":script": &txout.script_pubkey.0,
         ":value_zat": &i64::from(Amount::from(txout.value)),
-        ":height": output_height,
+        ":max_observed_unspent_height": max_observed_unspent.map(u32::from),
     ];
 
     let utxo_id = stmt_upsert_transparent_output
         .query_row(sql_args, |row| row.get::<_, i64>(0).map(UtxoId))?;
     Ok(utxo_id)
 }
 
+pub(crate) fn queue_transparent_spend_detection<P: consensus::Parameters>(
+    conn: &rusqlite::Transaction<'_>,
+    params: &P,
+    receiving_address: TransparentAddress,
+    tx_ref: TxRef,
+    output_index: u32,
+) -> Result<(), SqliteClientError> {
+    // Add an entry to the transaction retrieval queue if we don't already have raw transaction
+    // data.
```

Comment:
```suggestion
    // Add an entry to the transaction retrieval queue if we don't already
    // have one for this txout.
```
This doesn't optimize the case where we already have the raw transaction data. Should it?

Reply:
Yes, good catch.

Reply:
Oh, wait, no; this isn't a request for specific transaction data.